app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -10,6 +10,9 @@ class GetUspsProofingResultsJob < ApplicationJob
     "State driver's license",
     "State non-driver's identification card",
   ]
+  SUPPORTED_SECONDARY_ID_TYPES = [
```

Comment:
how do we feel about changing `types` to something like `reasons` to make the naming more descriptive of what's stored in the array?

Reply:
Even though this particular value for the field is unconventional, it is labeled `secondaryIdType` by the USPS API, so I think this name is appropriate.

Reply:
ok that makes sense then